src/operators/IntegralCurve/avtIntegralCurveFilter.C

```diff
@@ -3202,6 +3232,89 @@ avtIntegralCurveFilter::ComputeCorrelationDistance(int idx,
     return val;
 }
 
+// ****************************************************************************
+// Method:  avtIntegralCurveFilter::CheckForClosedCurve
+//
+// Purpose: Compute the correlation distance at this point. Defined as
+//   the arc length distance from the current point to the next point
+//   (greater than minDist away) along the streamilne where the
+//   velocity direction is the same (to angTol).
+//
+// Arguments:
+//   
+//
+// Programmer:  Dave Pugmire
```

Comment:
Is function header a cut-n-pasted from somewhere else? You, not D. Pugmire wrote this, correct? Maybe ensure the Purpose comment is correct too?

Reply:
Yeah Dave would not credit for my crappy code. I edited the comments and checked in a revised version.